modules/storages/app/workers/storages/health_status_mailer_job.rb

```diff
@@ -40,14 +40,15 @@ class HealthStatusMailerJob < ApplicationJob
     )
 
     def perform(storage:)
-      return unless Storages::Storage.exists?(storage.id)
+      return unless ::Storages::Storage.exists?(storage.id)
 
       storage.reload
```

Comment:
```suggestion
```
The reload is unnecessary, because the storage has just been loaded from the database. It is not related to your changes as well. But you could just remove it I think.

Reply:
Why was the storage just loaded from the database, when the job got dispatched 12h ago?

Reply:
When we use ActiveRecord instance as a parameter then https://github.com/rails/globalid is used to serialize this parameter. Then in the database there is something like: `gid://app/Storage/1`. Then when the parameter is deserialized it has been fetched from the database and modal instance has been created with actual data. So, because of that behavior we do need to reload at the beginning of `perform` method.